bc_obps/registration/schema/operation.py

```diff
@@ -77,6 +72,8 @@ class Config:
             "id",  # need to exclude id since it's auto generated and we don't want to pass it in
             "documents",  # excluding documents because they are handled by individual form fields
             *AUDIT_FIELDS,
+            "status",
+            "submission_date",
```

Comment:
Since there are lots of fields that we don't want to handle while updating an operation (to name some of them:           `swrs_facility_id`, `bcghg_id`, `operation_has_multiple_operators`, `verified_at`, `verified_by`, `bc_obps_regulated_operation`, `reporting_activities`, `point_of_contact_id`), I suggest replace `model_exclude` with `model_fields` and only specify the fields that are required for updating and we care about them.